src/main/java/com/eleks/academy/whoami/controller/GameController.java

```diff
@@ -23,76 +24,73 @@
-	}
-
-	@PostMapping
-	@ResponseStatus(HttpStatus.CREATED)
-	public GameDetails createGame(@RequestHeader(PLAYER) String player,
-								  @Valid @RequestBody NewGameRequest gameRequest) {
-		return this.gameService.createGame(player, gameRequest);
-	}
-
-	@GetMapping("/{id}")
-	public ResponseEntity<GameDetails> findById(@PathVariable("id") String id,
-												@RequestHeader(PLAYER) String player) {
-		return this.gameService.findByIdAndPlayer(id, player)
-				.map(ResponseEntity::ok)
-				.orElseGet(() -> ResponseEntity.notFound().build());
-	}
-
-
-	@PostMapping("/{id}/players")
-	public SynchronousPlayer enrollToGame(@PathVariable("id") String id,
-										  @RequestHeader(PLAYER) String player) {
-		return this.gameService.enrollToGame(id, player);
-	}
-
-	@PostMapping("/{id}/characters")
-	@ResponseStatus(HttpStatus.OK)
-	public void suggestCharacter(@PathVariable("id") String id,
-								 @RequestHeader(PLAYER) String player,
-								 @Valid @RequestBody CharacterSuggestion suggestion) {
-		this.gameService.suggestCharacter(id, player, suggestion);
-	}
-
-	@GetMapping("/{id}/turn")
-	public ResponseEntity<TurnDetails> findTurnInfo(@PathVariable("id") String id,
-													@RequestHeader(PLAYER) String player) {
-		return this.gameService.findTurnInfo(id, player)
-				.map(ResponseEntity::ok)
-				.orElseGet(() -> ResponseEntity.notFound().build());
-	}
-
-	@PostMapping("/{id}")
-	public ResponseEntity<GameDetails> startGame(@PathVariable("id") String id,
-												 @RequestHeader(PLAYER) String player) {
-		return this.gameService.startGame(id, player)
-				.map(ResponseEntity::ok)
-				.orElseGet(() -> ResponseEntity.notFound().build());
-	}
-
-	@PostMapping("/{id}/questions")
-	public void askQuestion(@PathVariable("id") String id,
-							@RequestHeader(PLAYER) String player, @RequestBody Message message) {
-		this.gameService.askQuestion(id, player, message.getMessage());
-	}
-
-	@PostMapping("/{id}/guess")
-	public void submitGuess(@PathVariable("id") String id,
-							@RequestHeader(PLAYER) String player, @RequestBody Message message) {
-		this.gameService.submitGuess(id, player, message.getMessage());
-	}
-
-	@PostMapping("/{id}/answer")
-	public void answerQuestion(@PathVariable("id") String id,
-							   @RequestHeader(PLAYER) String player, @RequestBody Message message) {
-		this.gameService.answerQuestion(id, player, message.getMessage());
-
-	}
+    private final GameService gameService;
+
+    @GetMapping
+    public List<PersistentGame> findAvailableGames(@RequestHeader(PLAYER) String player) {
+        return this.gameService.findAvailableGames();
+    }
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public ResponseEntity<GameDetails> createGame(@RequestHeader(PLAYER) String player,
+                                                  @Valid @RequestBody NewGameRequest gameRequest) {
+        return ResponseEntity.ok(this.gameService.createGame(player, gameRequest));
```

Comment:
1) ResponseEntity.ok() is not needed here
2) `ResonseEntity.ok()` is 200 status while you declare the `@ResponseStatus(201)`